src/init.cpp

```diff
@@ -1763,7 +1763,7 @@ bool AppInit2(const std::vector<std::string>& words)
 
         if (!zwalletMain->GetMasterSeed().IsNull()) {
             //Inititalize zPIVWallet
```

Comment:
Should this be changed to zZNZWallet?

Reply:
Yup

Reply:
Actually, just re-ran the compiler and it's all fine (I did this yesterday, but double-checking).
Since this is Zerocoin code (And there's a bunch of PIV branded zerocoin code still), it may be better to just leave it, as we will be removing it all anyway in another PR, 99% of our Zerocoin code will be purged soon.